MiniBootcamp/MiniBootcampTests/TweetModelTests.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  TweetModelTests.swift
+//  MiniBootcampTests
+//
+//  Created by Chioma Amanda Mmegwa  on 29/05/2023.
+//
+
+import XCTest
+@testable import MiniBootcamp
+
+final class TweetModelTests: XCTestCase {
+    func test_mapToViewModel() {
+      let sut = Tweet(
+        user: User(
+          followersCount: 20,
+          name: "test",
+          screenName: "screenName",
+          description: "description",
+          location: "location",
+          createdAt: "createdAt",
+          profileBackgroundColor: "backgroundColor",
+          profileImageURL: "imageUrl",
+          profileBackgroundImageURL: "backgroundImageUrl"
+        ),
```

Comment:
For readability, declare the user property apart and then use it to build the tweet struct.